src/element.rs

```diff
@@ -46,3 +46,15 @@ implement!(usize);
 
 implement!(c32, c32::new(0.0, 0.0));
 implement!(c64, c64::new(0.0, 0.0));
+
+impl<T> Element for Option<T> where T: Copy + PartialEq {
```

Comment:
My mistake--I'd actually like to update the description, as for many of the applications I'm using `Option<T>` matrix elements, their notion of zero is the absence of data.